### PR TITLE
Initial working implementation

### PR DESCRIPTION
**Pull request by matt-peters**

### CODE REVIEW

**Comment by matt-peters** [comment]
The travis build is failing because it's timing out when installing scipy, I'll fix that before merging.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,136 @@
...
+represents each word as a multivariate Gaussian.  Assuming some dictionary
+of known tokens `w[i], i = 0 .. N-1`, each word is represented as
+a probability `P[i]`, a `K` dimensional Gaussian parameterized by
+```
+   P[i] ~ N(x; mu[i], Sigma[i])
+```
+Here, `mu[i]` and `Sigma[i]` are the mean and co-variance matrix
+for word `i`.  The mean is a vector of length `K` and in the most general
+case `Sigma[i]` is a `(K, K)` matrix.  The paper makes one of two
+approximations to simply `Sigma[i]`:
```
Comment (ghost) [inline README.md]: 'simply' -> 'simplify'

File: word2gauss/embeddings.pyx [hunk 2]
```diff
@@ -0,0 +1,944 @@
...
+        Writes a gzipped text file of the model in word2vec text format
+
+        word_or_id1 0.5 -0.2 1.0 ...
+        word_or_id2 ...
+
+        This class doesn't have knowledge of id -> word mapping, so
+        it can be passed in as a callable vocab(id) return the word string
+
+        if full=True, then writes out the full model.  It is a tar.gz
+        file the word vectors and Sigma files, in addition
```
Comment (ghost) [inline word2gauss/embeddings.pyx]: Missing a word here? 'file the word vectors...'  -> 'file containing the word vectors...'

File: README.md [hunk 3]
```diff
@@ -1,2 +1,136 @@
...
+Then `make install`.  `make test` runs the test suite.
+
+### Code overview
+The `GaussianEmbedding` class is the main workhorse for most tasks.  It
+stores the model data, deals with serialization to/from files and
+learns the parameters.  To allow embedding of non-word types like
+hierarchies and entailment relations, `GaussianEmbedding` has
+no knowledge of any vocabulary and operates only on
+unique IDs.  Each ID is a `uint32` from `0 .. N-1` with `-1` signifying
+an OOV token.
```
Comment (b4hand) [inline README.md:23]: I'm sure OOV may be a well-known acronym inside the NLP community, but I would probably expand this out at least on its first usage. I'm assuming this means "out-of-vocabulary".

**Pull request merged**